plugins/transport-grpc/src/core/create-grpc-client-options.ts

```diff
@@ -0,0 +1,63 @@
+import { builders, namedTypes } from "ast-types";
+import { interpolate } from "../util/ast";
+import { join } from "path";
+import { templatesPath } from "../constants";
+import {
+  readFile,
+  removeTSClassDeclares,
+  print,
+} from "@amplication/code-gen-utils";
+import { DsgContext, Module } from "@amplication/code-gen-types";
+
+const grpcClientOptionsPath = join(
+  templatesPath,
+  "grpc.client.options.template.ts"
+);
+const fileName = "grpc.client.options.ts";
+
+export async function createGrpcClientOptionsFile(
+  context: DsgContext
+): Promise<Module> {
+  const template = await readFile(grpcClientOptionsPath);
+  const { serverDirectories, entities } = context;
+
+  let packages: namedTypes.ArrayExpression = {
+    elements: [],
+    type: "ArrayExpression",
+  };
+  let protoPaths: namedTypes.ArrayExpression = {
+    elements: [],
+    type: "ArrayExpression",
+  };
+
+  entities?.forEach((entity) => {
+    const entityNameLowerCase = entity.name.toLowerCase();
+    packages.elements.push(builders.stringLiteral(entityNameLowerCase));
+    const entityProtoPath = builders.stringLiteral(
+      `src/${entityNameLowerCase}/${entityNameLowerCase}.proto`
+    );
+
+    protoPaths.elements.push(entityProtoPath);
+  });
+
+  const packagesArray = packages;
+
+  const protoPathArray = protoPaths;
+
+  const templateMapping = {
+    PACKAGES_ARRAY: packagesArray,
+    PROTO_PATH_ARRAY: protoPathArray,
+    PORT_PATH: builders.stringLiteral("localhost:9090")
```

Comment:
Should this be hard-coded here ("localhost:9090") and not arrive from the plugin settings?

Reply:
@mulygottlieb This is the default value for the template mapping.
 In the file, we are using the value from the env file:

<img width="668" alt="image" src="https://github.com/amplication/plugins/assets/97830649/bd4fe0ee-a9e6-4a7a-9d5a-1654801ed0f5">